scripts/release-post/index.mjs

```diff
@@ -28,7 +28,7 @@ import handlebars from 'handlebars';
 import { format } from 'prettier';
 
 import { downloadsTable } from './downloadsTable.mjs';
-import prettierConfig from '../../.prettierrc.json' assert { type: 'json' };
+import prettierConfig from '../../.prettierrc.json' with { type: 'json' };
```

Comment:
We have import assertions in other places of the codebase, do you mind replacing those in this PR too?

Reply:
Sorry but I'm not interested in doing more than this.

Reply:
@nodejs/nodejs-website does someone have the bandwidth for that?

Reply:
on node 20 webpack's not happy but on 22 it's build correctly so let's do nothing for moment IMHO